src/frontend/src/lib/components/exchange/ExchangeBalance.svelte

```diff
@@ -17,7 +17,7 @@
 		{#if $exchangeInitialized}
 			{formatUSD(totalUsd)}
 		{:else}
-			&ZeroWidthSpace;
+			<span class="animate-pulse">{formatUSD(0)}</span>
```

Comment:
it is already above:

class={`break-all text-5xl font-bold ${totalUsd === 0 ? 'opacity-50' : 'opacity-100'} inline-block mt-8`}

I agree that is not the same condition, but they are linked since if not`$exchangeInitialized`, then `totalUsd` (that is calculated from `$combinedDerivedEnabledNetworkTokensUi`) will be zero.

The contrary is not true, fortunately.

but if you prefer, I can add it there either way, to be cleaner

Reply:
If already above, good. Thanks for the feedback.